_posts/2018-03-02-Kickstarter-Trajectory-2018-light.md

```diff
@@ -0,0 +1,183 @@
+These courses are created to give the beginning software developer a broad knowledge base while also providing an in-depth view on several technologies and best practices.
+
+This year the Kickstarter trajectory spanned 15 days, with topics ranging far and wide: backend to frontend, Spring Data JPA to TypeScript and everything in between.
+All of these courses will make sure the candidates will be able to hit the ground running on their first project as Ordina consultants.
+
+This post will summarize the training and experiences we've had while following the Kickstarter trajectory. 
+
+# Backlog
+
+<img src="/img/2018-03-02-Kickstarter-Trajectory-2018-light/kanban.jpg" style="max-width:100%"/>
+
+# What we have done
+
+### Git
+During the first days of our kickstart trajectory we've got a brief explanation about git by [Yannick De Turck](/author/yannick-de-turck/).
+He explained us that git is an open-source, distributed version control system that keeps track of your files and history.
+Basically this means git offers us tooling to collaborate on code bases without overwriting each others changes so easily.
+
+We saw which workflow JWorks uses in git and which commands we can use to do so.
+This way we learned how to create git repo's and create separate branches for features or different environments.
+And we even saw the different ways to merge these branches.
+One thing we'll definitely won't forget that fast is to rebase when pulling to keep a clean non-spaghetti like history,
+something that is preferred by many co-workers at JWorks! 
+At least that is what Yannick has told us ;)
+
+### Spring and Spring boot
+
+Those lectures were given by [Ken Coenen](/author/ken-coenen) and were spread over two days. 
+
+During the first lecture, we got a recap of the concepts of JPA, beans, application contexts and other things that Spring uses in its core fundamentals. Next, we dug deeper into the framework and introduced ourselves with Spring Web Services and Spring Security and created a small backend application during the second lecture. 
+
+Both lectures were rather theoretical, but very informative and elaborate with a lot of examples. So you have got everything you need to get familiar with Spring. You can find the course material on Ken's [Github](https://github.com/kencoenen/spring-course) page.
+
+### Microservices
+The workshop on microservices was lectured by [Kevin Van Houtte](/author/kevin-van-houtte/). 
+It went pretty fast and at the end of the workshop, 
+we had acquired a great overview of all the important aspects regarding the microservices architecture! 
+The most important thing to know is that each microservice takes care of only one specific business function.
+
+Currently, monolith architectures are still being used a lot within companies, 
+but as an up-to-date IT consultant it’s essential to know about microservices and where to use it.
```

Comment:
Indeed! Microservices are not a golden bullet and should only be considered if it fits the use case.